### DISCUSSION TITLE
Will /cmi lfix implement something like /paper fixlight on 1.16?

### DISCUSSION THREAD

**Discussion by variananora** [Zrips/CMI#3903] [Q&A]
**Question: **
Will /cmi lfix implement something like /paper fixlight on 1.16?

**Comment by CloudeLecaw** [comment]
What do you mean? Cmi lfix works fine o  paper too, whats your Problem with it?

> 
> lfix (range) (playerName) (stop/stopall)
> Fix light in chunks around you

Without any vars the amounzs in config are taken.
